<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>592</width>
    <height>327</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Logging</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>592</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuCurve">
    <property name="title">
     <string>Device</string>
    </property>
    <addaction name="actionDevice_connect"/>
    <addaction name="actionDisconnect"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuSetting">
    <property name="title">
     <string>Setting</string>
    </property>
    <addaction name="actionSetting_connect"/>
    <addaction name="actionParameter"/>
   </widget>
   <widget class="QMenu" name="menuLogging">
    <property name="title">
     <string>Logging</string>
    </property>
    <addaction name="actionCurve"/>
   </widget>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionSave_layout"/>
    <addaction name="actionLoad_layout"/>
    <addaction name="separator"/>
    <addaction name="actionExit_2"/>
   </widget>
   <widget class="QMenu" name="menuData">
    <property name="title">
     <string>Data</string>
    </property>
    <addaction name="actionLoad_data"/>
    <addaction name="actionExport_file"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuCurve"/>
   <addaction name="menuLogging"/>
   <addaction name="menuData"/>
   <addaction name="menuSetting"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionDevice_connect"/>
   <addaction name="actionDisconnect"/>
   <addaction name="actionSetting_connect"/>
  </widget>
  <action name="actionSetting_connect">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/images/settings.png</normaloff>:/images/images/settings.png</iconset>
   </property>
   <property name="text">
    <string>Connect setting</string>
   </property>
  </action>
  <action name="actionDevice_connect">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/images/connect.png</normaloff>:/images/images/connect.png</iconset>
   </property>
   <property name="text">
    <string>Connect</string>
   </property>
  </action>
  <action name="actionDisconnect">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/images/disconnect.png</normaloff>:/images/images/disconnect.png</iconset>
   </property>
   <property name="text">
    <string>Disconnect</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/images/application-exit.png</normaloff>:/images/images/application-exit.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionCurve">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/images/curve_attribute.png</normaloff>:/images/images/curve_attribute.png</iconset>
   </property>
   <property name="text">
    <string>Curve Attribute</string>
   </property>
  </action>
  <action name="actionSave_layout">
   <property name="text">
    <string>Save layout</string>
   </property>
  </action>
  <action name="actionLoad_layout">
   <property name="text">
    <string>Load layout</string>
   </property>
  </action>
  <action name="actionExit_2">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionLoad_data">
   <property name="text">
    <string>Load data</string>
   </property>
  </action>
  <action name="actionExport_file">
   <property name="text">
    <string>Export file</string>
   </property>
  </action>
  <action name="actionParameter">
   <property name="text">
    <string>Parameter</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
